deeplake/htype.py

```diff
@@ -144,6 +147,6 @@ def verify_htype_key_value(htype, key, value):
             raise TypeError(f"{key} must be of type {expected_type}, not {type(value)}")
         if expected_type == dict:
             expected_keys = set(htype_verifications[key].get("keys"))
-            present_keys = set(value.keys())
+            present_keys = set(value)
```

Comment:
what was the difference, why keys are removed?

Reply:
no difference